.github/CODEOWNERS

```
@@ -1 +1 @@
-@kalexand-rh @JoeAldinger @bergerhoffer @adellape @sjhala-ccs @apinnick @ousleyp
+@kalexand-rh @JoeAldinger @bergerhoffer @adellape @sjhala-ccs @apinnick @ousleyp @michaelryanpeter @mjpytlak @snarayan-redhat @opayne1 @kelbrown20 @bscott-rh @kcarmichael08 @JoeAldinger @jab-rh @rolfedh @abrennan89 @sjhala-ccs @abhatt-rh @gabriel-rh @Srivaralakshmi @bburt-rh
```

```suggestion
@kalexand-rh @JoeAldinger @bergerhoffer @adellape @sjhala-ccs @apinnick @ousleyp @michaelryanpeter @mjpytlak @snarayan-redhat @opayne1 @kelbrown20 @bscott-rh @kcarmichael08 @jab-rh @rolfedh @abrennan89 @sjhala-ccs @abhatt-rh @gabriel-rh @Srivaralakshmi @bburt-rh
```
Does Joe need to be in there twice?

---

No. Thank you!